assets/javascripts/discourse/templates/activity-pub-category-followers.hbs

```diff
@@ -0,0 +1,80 @@
+<Discovery::Navigation
+  @createTopic={{fn
+    this.composer.openNewTopic
+    (hash category=this.model.category preferDraft=true)
+  }}
+  @category={{this.model.category}}
+/>
+
+<ActivityPubCategoryBanner @category={{this.model.category}} />
+
+<LoadMore
+  @class="activity-pub-followers-container"
+  @selector=".directory-table .directory-table__cell"
+  @action={{action "loadMore"}}
+>
+  {{#if this.model.hasFollowers}}
+    <ResponsiveTable @className="activity-pub-followers">
+      <:header>
+        <TableHeaderToggle
+          @field="actor"
+          @labelKey="discourse_activity_pub.follower.actor"
+          @automatic={{true}}
+          @order={{this.order}}
+          @asc={{this.asc}}
+        />
+        <TableHeaderToggle
+          @field="user"
+          @labelKey="discourse_activity_pub.follower.user"
+          @automatic={{true}}
+          @order={{this.order}}
+          @asc={{this.asc}}
+        />
+        <TableHeaderToggle
+          @field="followed_at"
+          @labelKey="discourse_activity_pub.follower.followed_at"
+          @automatic={{true}}
+          @order={{this.order}}
+          @asc={{this.asc}}
+        />
+      </:header>
+      <:body>
+      {{#each this.model.followers as |follower|}}
+        <div class="directory-table__row activity-pub-follower">
```

Comment:
Maybe each row in this loop can be its own component, that might help avoid having to use helpers for the image and the handle.

Reply:
Personally I prefer to not do this with tables as I feel it makes the table structure a bit confusing, but happy to do it if you think it makes more sense Discourse-coding-standards-wise.

Reply:
It's fine as is, thanks.